tools/kete.py

```diff
@@ -706,6 +756,10 @@ def __init__(self, completekey='tab', stdin=None, stdout=None):
         except FileExistsError:
             pass
 
+        self._config_file = os.path.join(CONFIG_PATH, 'settings.ini')
+        self._config = configparser.ConfigParser()
+        self._config.read(self._config_file)
```

Comment:
I wonder if we should not create a basic `settings.ini` when opening `kete` so we can document the various values that we are going to use.

Something like `if not os.path.exists(self._config_file)`